wp-content/plugins/ig-revisions/plugin.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Plugin Name: Integreat Page Revisions
+ * Description: Set a specific page revision for delivery via API
+ * Version: 1.0
+ * Author: Sven Seeberg
+ * Author URI: https://github.com/Integreat
+ * License: MIT
+ * Text Domain: ig-revisions
+ */
+
+function ig_revisions_metabox( $post ) {
+	$children = wp_get_post_revisions( $post_id );
+	$revision_id = get_post_meta( $post->ID, 'ig_revision_id', true );
+	$options = "<option value='-1'" . (!$revision_id ? " selected" : "") . ">None</option>";
+	
+	foreach($children as $child) {
+		$options .= "<option value='" . $child->ID . "'" . ($child->ID==$revision_ids ? " selected" : "" ) . ">" . $child->post_date . "</option>";
+	}
+	add_meta_box(
+		'ig_revisions',
+		__('Set Revision', 'ig_revisions'),
+		'ig_revisions_metabox_html',
+		null,
+		'advanced',
+		'default',
+		$options
+	);
+}
+add_action( 'add_meta_boxes_post', 'ig_revisions_metabox' );
```

Comment:
At the moment the custom meta box is only shown when editing posts and is not available when editing pages and the disclaimer, so I'd propose using `add_action( 'add_meta_boxes', 'ig_revisions_metabox' )` here.